API/public/javascripts/profile.js

```diff
@@ -76,7 +76,7 @@ function validatePassword() {
         }
 
         if (!hasLowerCase) {
-          $('.passwordError').text('Password must contain at least 1 lower case character!');
+          $('.passwordError').text('Password must contain at least lower case character!');
```

Comment:
Should we keep the `1` in?

Reply:
You are right. I'm not sure how I did not see that.

Reply:
Added a commit to fix it